ivadomed/__init__.py

```diff
@@ -1 +1,2 @@
 from .utils import __version__, __ivadomed_dir__
+from .random import set_seed
```

Comment:
I usually do not recommend relative imports, unless absolutely necessary. A good explanation is provided here: https://realpython.com/absolute-vs-relative-python-imports/#pros-and-cons-of-relative-imports

Reply:
Okay. I will remedy that. I was following the existing convention here.

Reply:
Totally understandable. I was in the same shoes. There were some dated practices that I see around that could be improved and I found myself frequently introduce them in PRs and bloated PR to large size than usual. I think you have many great ideas and feel free to experiment them. I found that what helps me the most is do some trailblazer branches and then look back on how that can be brokeninto smaller/modular PRs to make the incorporate easier to adopt. I can't say I mastered that myself but I find that smaller PR proportionally increase approval time magically somehow. 👍 . Keep up the good recommendations.

Reply:
Thank you, and advice well taken note of.